app/src/main/java/com/example/splendormobilegame/activities/GameActivity/GameActivity.java

```diff
@@ -157,6 +158,11 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
                 ServerMessageType.NEW_ROOM_OWNER,
                 this.newRoomOwnerController.getNewRoomOwnerResponse()
         );
+
+        CustomWebSocketClient.getInstance().assignReactionToMessageType(
+                ServerMessageType.NOBLE_RECEIVED_ANNOUNCEMENT,
+                this.nobleController.getNobleReceivedResponse()
+        );
```

Comment:
Instance of `nobleController` is not created in the `onCreate` method, but it should like others controllers.